index.bs

```diff
@@ -295,23 +506,34 @@ spec: webidl
           descriptor.
         </p>
         <p>
-          If the descriptor does not have a {{deviceId}}, its semantic is that
-          it queries for access to all devices of that class. Thus, if a
-          query for the {{"camera"}} permission with no {{deviceId}} returns
-          {{"granted"}}, the client knows that there will never be a permission
-          prompt for a camera, and if {{"denied"}} is returned, it knows that
-          no getUserMedia request for a camera will succeed.
+          If the descriptor does not have a
+          {{DevicePermissionDescriptor/deviceId}}, its semantic is that it
+          queries for access to all devices of that class. Thus, if a query for
+          the {{"camera"}} permission with no
+          {{DevicePermissionDescriptor/deviceId}} returns {{"granted"}}, the
+          client knows that there will never be a permission prompt for a
+          camera, and if {{"denied"}} is returned, it knows that no getUserMedia
+          request for a camera will succeed.
         </p>
         <p>
           If a permission state is present for access to some, but not all,
-          cameras, a query without the {{deviceId}} will return {{"prompt"}}.
+          cameras, a query without the {{DevicePermissionDescriptor/deviceId}}
+          will return {{"prompt"}}.
+        </p>
+        <p class="issue" id="issue-media-granted">
+          It may not make sense for `{name: "camera"}`'s <a>permission state</a>
+          to ever be {{"granted"}}: if the site calls
+          {{MediaDevices/getUserMedia()}} with constraints that none of the
+          user's devices satisfy, the UA cannot return a device without
+          prompting, but saying {{"granted"}} promised to do so.
```

Comment:
I'm having a hard time parsing this paragraph. I don't understand "but saying {{"granted"}} promised to do so." Maybe change it to active voice? promised by who?